code/ai/aigoals.h

```diff
@@ -152,6 +152,8 @@ typedef struct ai_goal {
 
 extern void ai_goal_reset(ai_goal *aigp, bool adding_goal = false, ai_goal_mode ai_mode = AI_GOAL_NONE, int ai_submode = -1, ai_goal_type type = ai_goal_type::INVALID);
 
+// Reset all path points. Used in the ship lab. Missions clean up path points with the garbage collector.
```

Comment:
What do you mean by garbage collector? Is there a container holding path point objects that handles destruction on mission end?